MAUI/PDF-Viewer/Scrolling.md

```diff
@@ -0,0 +1,107 @@
+        {
+            SfPdfViewer PdfViewer = new SfPdfViewer();
+            // Subscribe to property changed event.
+            PdfViewer.PropertyChanged += PdfViewer_PropertyChanged;
+        }
+
+{% endhighlight %}
+{% endtabs %}
+
+Handle scroll changes by implementing the event handler method. The below code example illustrates how to respond to both horizontal and vertical scroll changes:
+
+{% tabs %}
+{% highlight c# %}
+        private void PdfViewer_PropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
+        {
+            if (e.PropertyName == nameof(SfPdfViewer.HorizontalOffset))
+            {
+                // Horizontal scroll change detected
+                // Your code here
+            }
+            else if (e.PropertyName == nameof(SfPdfViewer.VerticalOffset))
+            {
+                // Vertical scroll change detected
+                // Your code here
+            }
+        }
+{% endhighlight %}
+{% endtabs %}
+
+## Detecting the end of the document
+
+You can determine if the control has reached the vertical end of the document by evaluating the following properties:
+
+* `VerticalOffset`: It provide the current vertically scrolled offset.
+* `ClientRectangle`: It provides the viewport area bounds.
+* `ExtentHeight`: It provides the overall vertical height of the PDF document.
+
+Below is the code example to detect whether the control has reached the vertical end of the document.
+
+{% tabs %}
+{% highlight c# %}
+        private void PdfViewer_PropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
+        {
+            if (e.PropertyName == nameof(SfPdfViewer.VerticalOffset))
+            {
+                // Determine if the control has reached the vertical end of the document.
+                bool isVerticalEndReached = (PdfViewer.VerticalOffset + PdfViewer.ClientRectangle.Height >= PdfViewer.ExtentHeight);
+            }
+        }
+{% endhighlight %}
+{% endtabs %}
+
+Similarly for horizontal end detection, refer to the below code example.
+
+{% tabs %}
+{% highlight c# %}
+        private void PdfViewer_PropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
+        {
+            if (e.PropertyName == nameof(SfPdfViewer.HorizontalOffset))
+            {
+                // Determine if the control has reached the horizontal end of the document.
+                bool isHorizontalEndReached = (PdfViewer.HorizontalOffset + PdfViewer.ClientRectangle.Width >= PdfViewer.ExtentWidth);
+            }
+        }
+{% endhighlight %}
+{% endtabs %}
+
+W> Please note that since the PDF Viewer has built-in scrolling capability, it is advised avoid placing the PDF Viewer inside other controls that also offer scrolling, such as [ScrollView](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/scrollview?view=net-maui-8.0). Nesting within such controls may cause unexpected issues.
```

Comment:
W> Please note that since the PDF Viewer has built-in scrolling capability, it is advised to avoid placing the PDF Viewer inside other controls that also offer scrolling, such as [ScrollView](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/scrollview?view=net-maui-8.0). Nesting within such controls may cause unexpected issues.